native-code/android/index.md

```diff
@@ -77,6 +77,34 @@ For instructions on how to build and run, see
 [webrtc/examples/androidapp/README][3].
 
 
+### Using Android Studio
+
+  1. Build the project normally:
+
+     ~~~~~ bash
+     ninja -C out/Debug AppRTCMobile
+     ~~~~~
+
+  2. Generate the project files:
+
+     ~~~~~ bash
+     build/android/gradle/generate_gradle.py --output-directory $PWD/out/Debug \
+     --target "//webrtc/examples:AppRTCMobile" --use-gradle-process-resources \
+     --split-projects
+     ~~~~~
+
+  3. *Import* the project in Android Studio. (Do not just open it.) The project
+     is located in `out/Debug/gradle`. If asked which SDK to use, choose to use
+     Android Studio's SDK. When asked whether to use the Gradle wrapper, press
+     "OK".
+
+  4. Ensure target `webrtc>examples>AppRTCMobile` is selected and press Run.
+     AppRTCMobile should now start on the device.
+
+If you do any changes to the C++ code, you have to compile the project using
+ninja after the changes (see step 1).
```

Comment:
Does one need to do any kind of reloading in Android Studio after recompiling or will it automatically pick up the updated library?